lib/talon/concern.ex

```diff
@@ -306,6 +306,7 @@ defmodule Talon.Concern do
   Note: This function is overridable
   """
   @spec nav_action_links(atom, atom, Struct.t | Module.t) :: List.t
+  def nav_action_links(_concern, _action, _resource = nil), do: []
```

Comment:
I don't think `_resource =` is necessary here. if you fell strongly, then I would prefer it be written `nil = _resource`